x-pack/plugins/enterprise_search/server/lib/route_config_helpers.ts

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { RouteConfig, RouteConfigOptionsBody, RouteMethod } from 'kibana/server';
+
+import { schema } from '@kbn/config-schema';
+
+/**
+ * Kibana Enterprise Search Plugin API endpoints often times pass through the request
+ * body to the Enterprise Search API endpoints for validation. In those cases, we do not
+ * need to validate them in Kibana.
+ *
+ * The safe way to do that is to turn off body parsing entirely using `options.body.parse: false`.
+ * The will pass a String Buffer to the route handler. The proper way to validate this when validation
+ * is enabled to to use `body: schema.buffer()`.
```

Comment:
This took me a weirdly long time to find the Kibana docs for these options 😅 Can we link to them for posterity here?

```suggestion
 * The safe way to do that is to turn off body parsing entirely using `options.body.parse: false`.
 * The will pass a String Buffer to the route handler. The proper way to validate this when validation
 * is enabled to to use `body: schema.buffer()`.
 *
 * @see https://github.com/elastic/kibana/blob/master/docs/development/core/server/kibana-plugin-core-server.routeconfigoptionsbody.md
 * @see https://github.com/elastic/kibana/blob/master/packages/kbn-config-schema/README.md#schemabuffer
```